templates/pioneer-sgsb-beyond-the-paywall/page-nav-sensitive/page-nav-sensitive.1.schema.json

```diff
@@ -45,6 +49,7 @@
     "userId",
     "domain",
     "visitDuration",
+    "attentionDuration",
```

Comment:
Are there still documents flowing that do not have the new `attentionDuration` field? If so, we should not add this field to the `required` list, since that will cause us to start rejecting all submissions that don't have this field.

Reply:
Because we can't guarantee timing of associated add-on updates (or if clients update at all), the new field not marked as required anymore.